RetoEuskalmet/src/test/java/modelo/TestRealicionExiste.java

```diff
@@ -0,0 +1,27 @@
+package modelo;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class TestRealicionExiste {
+	private String Municipios = "Basauri";
+	private String NomEsNat = "Pantano";
+	RelacionExiste Vacio = new RelacionExiste();
+	
+	RelacionExiste RE = new RelacionExiste(Municipios,NomEsNat);
+	@Test
+	public void RelacionExisteConstructorLlenoTest() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.